### PR TITLE
Introduce PropertyReadMap and PropertyWriteMap in TS

### PR DESCRIPTION
**Pull request by relu91**
Synchronize Typescript types with the latest changes in #305

### CODE REVIEW

File: typescript/index.d.ts [hunk 1]
```diff
@@ -180,7 +180,8 @@ declare namespace WoT {
         data?: any;
     }
 
-    export type PropertyMap = object | { [key: string]: any; };
+    export type PropertyReadMap = Map<string,InteractionOutput>
+    export type PropertyWriteMap = Map<string,InteractionInput>
```
Comment (danielpeintner) [inline typescript/index.d.ts:184]: nits:
* spaces between arguments -> `Map<string, InteractionOutput>`
* `;` in the end to be consistent with the rest?

**Review by danielpeintner — APPROVED** [review]
maybe doing first some consistency updates

**Comment by danielpeintner** [comment]
Scripting Call 2021-03-01:
* content-wise the changes look ok
* @relu91 will update the minor formatting and we can merge

File: typescript/index.d.ts (hunk 1 above)
Comment (relu91) [reply typescript/index.d.ts]: Should be solved now

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**